devlib/utils/types.py

```diff
@@ -68,6 +68,15 @@ def numeric(value):
     """
     if isinstance(value, int):
         return value
+
+    if isinstance(value, basestring):
+        value = value.strip()
+        if value.endswith('%'):
+            try:
+                return float(value.rstrip('%')) / 100
```

Comment:
My bad, I didn't think carefully. However we can probably refactor that a little bit using `ast.eval_literal()`. What about something like:
```python
if isinstance(value, int) or isinstance(value, float):
    return value
if isinstance(value, basestring):
    try:
        return ast.eval_literal(value)
    except:
        pass
    if value.endswith('%'):
        cvalue = value.rstrip('%')
        try:
            return ast.eval_literal(cvalue) / 100.0
        except:
            pass # Don't raise excpetion here to extend more easily with other value.endswith('whatever')
raise ValueError('Not numeric: {}'.format(value))
```